pom.xml

```diff
@@ -68,10 +68,10 @@
         <commons-collections.version>3.2.2</commons-collections.version>
         <commons-lang3.version>3.10</commons-lang3.version>
         <curator.version>2.12.0</curator.version>
-        <delta.version>0.8.0</delta.version>
+        <delta.version>1.0.0</delta.version>
```

Comment:
is this related？

Reply:
spark 3.1 needs delta 1.0.0

Reply:
does delta 1.0.0 drop spark 3.0 support？

Reply:
Yes, see https://github.com/delta-io/delta/issues/690

Reply:
so this should be added to our 3.0 profile

Reply:
it's already there :)